recruiter/tests.py

```diff
@@ -20,6 +22,40 @@ def example_recruiter(example_user):
     return recruiter
 
 
+@pytest.fixture
+def bad_recruiter_example():
```

Comment:
this name is confusing as you're not creating any recruiter as the fixture name states. I'd change it to something like `invalid_recruiters_data`.

Reply:
will be fixed